packages/loader/src/resource-deserialize/resources/parser/ParserContext.ts

```
@@ -1,23 +1,20 @@
- * @class ParserContext
- * @template T
- * @template I
+ * @internal
  */
 export class ParserContext<T extends IHierarchyFile, I extends EngineObject> {
   entityMap: Map<string, Entity> = new Map();
   entityConfigMap: Map<string, IEntity> = new Map();
   components: Map<string, Component> = new Map();
   rootIds: string[] = [];
   strippedIds: string[] = [];
+  componentWaitingMap: Map<string, Function[]> = new Map();
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)